### PR TITLE
Update notebook screenshots

### LINKED ISSUE #2264: [BUG] Jupyter notebook App screenshots need some cleanup
On `fiftyone==0.18`, using `session.freeze()` to generate a screenshot in the App needs some cleanup:
- Selection boxes shouldn't appear
- Shadows should not appear
- Grid zoom tooltip looks weird

Screen Shot 2022-11-08 at 6 33 05 PM

Screen Shot 2022-11-08 at 6 33 38 PM

### LINKED ISSUE #2364: [BUG] 3d visualizer doesn't show upon generating docs from jupyter notebook
@ritch @benjaminpkane

### Commands to reproduce

in notebook in docs/source/tutorials:
`import fiftyone as fo`
`import fiftyone.zoo as foz`
`dataset = foz.load_zoo_dataset("kitti-multiview")`
`session = fo.launch_app(dataset)`
`session.freeze()`

Then run `bash docs/generate_docs.bash` in terminal.

Attaching a screenshot. On the left is the "left" image. On the right should be the 3d visualizer. It appears without problem in the notebook, but turns blank upon generating html docs...

Screenshot 2022-11-28 at 5 35 56 PM

### What areas of FiftyOne does this bug affect?

-   [X] `App`: FiftyOne application issue
-   [ ] `Core`: Core Python library issue
-   [ ] `Server`: FiftyOne server issue

### PR DESCRIPTION
**Pull request by benjaminpkane**
Resolves #2264 
Resolves #2364

Screen Shot 2022-12-05 at 8 28 17 PM

### CODE REVIEW

**Review by brimoor — CHANGES_REQUESTED** [review]
Most of the issues from #2264 are fixed, but I'm still seeing the checkboxes in the grid after freezing:
Screen Shot 2022-12-05 at 10 39 06 PM

I do also now get the following error when trying to open the expanded modal for a grouped dataset on this branch:

```
index.01da734e.js:134 TypeError: Cannot read properties of undefined (reading 'group')
    at thumbnailTitle (index.01da734e.js:1700:2582)
    at z4.renderSelf (index.01da734e.js:750:10502)
    at z4.render (index.01da734e.js:744:297578)
    at index.01da734e.js:744:297668
    at Array.forEach (<anonymous>)
    at X4.render (index.01da734e.js:744:297615)
    at Xst.updater (index.01da734e.js:755:59732)
    at Xst.attach (index.01da734e.js:755:62445)
    at ete.render (index.01da734e.js:1700:3592)
    at index.01da734e.js:1284:3310
```

**Comment by brimoor** [comment]
Is it straightforward to include the map and Looker3D in the screenshots? https://github.com/voxel51/fiftyone/issues/2264#issuecomment-1338700054

@jacobmarks has a 3D tutorial that he'd like to publish, but is currently blocked because Looker3D is absent from the frozen App images.

**Comment by brimoor** [comment]
@benjaminpkane Actually looks like the grouped dataset modal error is coming from `develop`, not this branch. Seeing it there too.

**Comment by benjaminpkane** [comment]
Unrelated, merged develop.
> ```
> index.01da734e.js:134 TypeError: Cannot read properties of undefined (reading 'group')
>     at thumbnailTitle (index.01da734e.js:1700:2582)
>     at z4.renderSelf (index.01da734e.js:750:10502)
>     at z4.render (index.01da734e.js:744:297578)
>     at index.01da734e.js:744:297668
>     at Array.forEach (<anonymous>)
>     at X4.render (index.01da734e.js:744:297615)
>     at Xst.updater (index.01da734e.js:755:59732)
>     at Xst.attach (index.01da734e.js:755:62445)
>     at ete.render (index.01da734e.js:1700:3592)
>     at index.01da734e.js:1284:3310
> ```

**Comment by benjaminpkane** [comment]
> Most of the issues from #2264 are fixed, but I'm still seeing the checkboxes in the grid after freezing

Can you share steps? My screenshots do not have this issue

**Comment by brimoor** [comment]
I see the checkboxes just by doing this:

```py
import fiftyone as fo
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset("quickstart-groups")
session = fo.launch_app(dataset)
```

```py
session.freeze()
```

**Comment by brimoor** [comment]
@benjaminpkane confirmed that Looker3D now appears in screenshots, but the map doesn't yet.

**Comment by benjaminpkane** [comment]
WebGL is no captured for both

**Comment by benjaminpkane** [comment]
All things addressed

**Review by brimoor — APPROVED** [review]
LGTM, all types of screenshots are looking great now 💪  

Is there any low-hanging fruit to speed up the screenshot process? It can take 30 seconds on my machine to screenshot the grid view of a dataset. For reference, I find that it's typically quite a bit faster when the modal is open.

File: fiftyone/core/session/notebooks.py [hunk 1]
```diff
@@ -124,3 +125,53 @@ def capture(img: str, width: int) -> None:
         f"fiftyone.deactivate.{cell.subscription.replace('-', '_')}",
         lambda: client.send_event(fose.DeactivateNotebookCell()),
     )
+
+
+def display_databricks(
```
Comment (brimoor) [inline fiftyone/core/session/notebooks.py:130]: Remove databricks stuff for clarity?

**Comment by benjaminpkane** [comment]
[toDataURL](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL) has an optional image quality parameter which I haven tentatively set to 0.25. Example screenshot of quality 0.25 below

Screen Shot 2022-12-12 at 11 50 08 AM

**Pull request merged**